### PR TITLE
Phish pharm

### PR DESCRIPTION
**Pull request by violetcrestedwren**

### CODE REVIEW

File: components/ApiStatus.jsx [hunk 1]
```diff
@@ -12,27 +12,29 @@ type tPROPS = {
...
 
 
 const endpointLinkMap: Object = Object.freeze({
-  'foodevent': 'food/event',
-  'foodenforcement': 'food/enforcement',
+  'phishpharm': 'animaldrug/phishpharm',
   'drugevent': 'drug/event',
   'druglabel': 'drug/label',
+  'foodenforcement': 'food/enforcement',
+
```
Comment (dkrylovsb) [inline components/ApiStatus.jsx]: Food Event gone?

File: containers/EndpointStatusContainer.jsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ const EndpointStatusContainer = function (ComposedEndpointStatus: ReactClass): R
       if (!this.state.data) return <span />
 
       const path: string = this.props.path.replace(/(\/reference){1}/g, '')
-      const fullPath: string = `https://api.fda.gov${path}.json`
+      const fullPath: string = `http://ec2-54-86-220-160.compute-1.amazonaws.com:8000${path}.json`
```
Comment (dkrylovsb) [inline containers/EndpointStatusContainer.jsx]: Need to use Api.fda.gov

File: containers/InfographicContainer.jsx [hunk 3]
```diff
@@ -199,15 +199,16 @@ class InfographicContainer extends React.Component {
       // complete search field. with or without range
       const search: string = this._getFilterSearch(searchParam, range)
       // the entire query, api endpoint + search params + count params
-      const query: string = `https://api.fda.gov${endpoint}.json?${search}count=${countParam}`
-
+      //const query: string = `http://5c2d7184.ngrok.io${endpoint}.json?${search}count=${countParam}`
```
Comment (dkrylovsb) [inline containers/InfographicContainer.jsx]: Same here

File: containers/InfographicContainer.jsx [hunk 4]
```diff
@@ -199,15 +199,16 @@ class InfographicContainer extends React.Component {
...
+      const query: string = `http://ec2-54-86-220-160.compute-1.amazonaws.com:8000${endpoint}.json?${search}count=${countParam}`
       // results data is unfortunately not included when filtering
       // by count params, so we need to make an additional request
       // to update records only when searchParams change
 
       // the entire query, api endpoint + search params
       // we do not want to include count parameters when counting total records
-      const recordsQuery: string = `https://api.fda.gov${endpoint}.json?${search}`
+      //const recordsQuery: string = `http://5c2d7184.ngrok.io${endpoint}.json?${search}`
+      const recordsQuery: string = `http://ec2-54-86-220-160.compute-1.amazonaws.com:8000${endpoint}.json?${search}`
```
Comment (dkrylovsb) [inline containers/InfographicContainer.jsx]: Same

File: containers/InfographicContainer.jsx [hunk 5]
```diff
@@ -229,7 +230,7 @@ class InfographicContainer extends React.Component {
     // or make a request, check the meta of that request, and then
     // make another request with the date in the search field
     // i prefer to just make the download request (which gets cached anyway)
-    xhrGET('https://api.fda.gov/download.json', _handleDownloadResponse)
+    xhrGET('http://ec2-54-86-220-160.compute-1.amazonaws.com:8000/download.json', _handleDownloadResponse)
```
Comment (dkrylovsb) [inline containers/InfographicContainer.jsx]: Same

File: pages/animaldrug/phishpharm/_explorers.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+oneRecord:
+  params:
+    - search for all records with pub_date between 1992 and 1996.
+    - limit to 1 record.
+  query: 'http://ec2-54-86-220-160.compute-1.amazonaws.com:8000/animaldrug/phishpharm.json?search=pub_date=[1992+TO+1996]&limit=1'
```
Comment (dkrylovsb) [inline pages/animaldrug/phishpharm/_explorers.yaml]: Need to replace all refs to Build Server

Comment (beardedfinch) [inline pages/animaldrug/phishpharm/_explorers.yaml]: Fixed all of these.

File: pages/data/_content.yaml [hunk 7]
```diff
@@ -1,4 +1,10 @@
 -
+  url: /data/adae/
```
Comment (dkrylovsb) [inline pages/data/_content.yaml]: ADAE?

File: pages/data/adae/_meta.yaml [hunk 8]
```diff
@@ -0,0 +1,25 @@
+type: dataset
```
Comment (dkrylovsb) [inline pages/data/adae/_meta.yaml]: ADAEs will be pushed out separately

File: push-site.sh [hunk 9]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 TODAY=$(date +"%Y-%m-%d")
-BUCKET="open.fda.gov-$TODAY"
+BUCKET="open.fda.gov-phishpharm"
```
Comment (dkrylovsb) [inline push-site.sh]: Need to fix

**Comment by beardedfinch** [comment]
A&V changes needed in downloads/index.jsx.

**Comment by beardedfinch** [comment]
A&V changes are good to go.

**Pull request closed without merging**